system/events/attachments/types.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license
+ * This file is part of Bott.
+ *
+ * This project is dual-licensed:
+ * - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+ * - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+ *
+ * Copyright (C) 2025 DanielLaCos.se
+ */
+
+/**
+ * Image attachment types
+ */
+export enum BottImageAttachmentType {
+  JPEG = "image/jpeg",
+  PNG = "image/png",
+  GIF = "image/gif",
+  WEBP = "image/webp",
+}
+
+/**
+ * Text attachment types
+ */
+export enum BottTextAttachmentType {
+  TXT = "text/plain",
+  MD = "text/markdown",
+}
+
+/**
+ * Audio attachment types
+ */
+export enum BottAudioAttachmentType {
+  MP3 = "audio/mpeg",
+  OPUS = "audio/opus",
+  WAV = "audio/x-wav",
+}
+
+/**
+ * Video attachment types
+ */
+export enum BottVideoAttachmentType {
+  MP4 = "video/mp4",
+}
+
+/**
+ * Document attachment types
+ */
+export enum BottDocumentAttachmentType {
+  PDF = "application/pdf",
+  HTML = "text/html",
+}
+
+/**
+ * All attachment types
+ */
+export enum BottAttachmentType {
+  // Images
+  JPEG = BottImageAttachmentType.JPEG,
+  PNG = BottImageAttachmentType.PNG,
+  GIF = BottImageAttachmentType.GIF,
+  WEBP = BottImageAttachmentType.WEBP,
+  // Text
+  TXT = BottTextAttachmentType.TXT,
+  MD = BottTextAttachmentType.MD,
+  // Audio
+  MP3 = BottAudioAttachmentType.MP3,
+  OPUS = BottAudioAttachmentType.OPUS,
+  WAV = BottAudioAttachmentType.WAV,
+  // Video
+  MP4 = BottVideoAttachmentType.MP4,
+  // Documents
+  PDF = BottDocumentAttachmentType.PDF,
+  HTML = BottDocumentAttachmentType.HTML,
+}
+
+/**
+ * Array of image attachment types
+ */
+export const IMAGE_ATTACHMENT_TYPES = [
+  BottAttachmentType.JPEG,
+  BottAttachmentType.PNG,
+  BottAttachmentType.GIF,
+  BottAttachmentType.WEBP,
+] as const;
+
+/**
+ * Array of text attachment types
+ */
+export const TEXT_ATTACHMENT_TYPES = [
+  BottAttachmentType.TXT,
+  BottAttachmentType.MD,
+] as const;
+
+/**
+ * Array of audio attachment types
+ */
+export const AUDIO_ATTACHMENT_TYPES = [
+  BottAttachmentType.MP3,
+  BottAttachmentType.OPUS,
+  BottAttachmentType.WAV,
+] as const;
+
+/**
+ * Array of video attachment types
+ */
+export const VIDEO_ATTACHMENT_TYPES = [
+  BottAttachmentType.MP4,
+] as const;
+
+/**
+ * Array of document attachment types
+ */
+export const DOCUMENT_ATTACHMENT_TYPES = [
+  BottAttachmentType.PDF,
+  BottAttachmentType.HTML,
+] as const;
+
+// For backwards compatibility, export as BottEventAttachmentType
```

Comment:
No need to worry about backwards compatibility, we are unreleased: prefer concision for now